src/taraxad/main.cpp

```diff
@@ -10,60 +14,533 @@ using namespace std;
+  genesis_block_fields["author"] = "0x0000000000000000000000000000000000000000";
+  genesis_block_fields["timestamp"] = "0x5d422b80";
+
+  auto &state = final_chain["state"] = Json::Value(Json::objectValue);
+  state["disable_block_rewards"] = true;
+
+  auto &dpos = state["dpos"] = Json::Value(Json::objectValue);
+  dpos["deposit_delay"] = "0x5";
+  dpos["withdrawal_delay"] = "0x5";
+  dpos["eligibility_balance_threshold"] = "0xf4240";
+
+  auto &genesis_state = dpos["genesis_state"] = Json::Value(Json::objectValue);
+  auto delegate_from = "0x7e4aa664f71de4e9d0b4a6473d796372639bdcde";
+  auto &delegate_to = genesis_state[delegate_from] = Json::Value(Json::objectValue);
+  delegate_to["0x10ece4be76a0a2a6eb154f7e4b9503932fe7382f"] = "0xf4240";
+  delegate_to["0xa932a0fd90b395eaf827b1be428c3cd7b8069cf6"] = "0xf4240";
+  delegate_to["0xa3094c300310286e356cc14d3ecad2281f217462"] = "0xf4240";
+
+  auto &eth_chain_config = state["eth_chain_config"] = Json::Value(Json::objectValue);
+  eth_chain_config["byzantium_block"] = "0x0";
+  eth_chain_config["constantinople_block"] = "0x0";
+  eth_chain_config["dao_fork_block"] = "0xffffffffffffffff";
+  eth_chain_config["eip_150_block"] = "0x0";
+  eth_chain_config["eip_158_block"] = "0x0";
+  eth_chain_config["homestead_block"] = "0x0";
+  eth_chain_config["petersburg_block"] = "0x0";
+
+  auto &execution_options = state["execution_options"] = Json::Value(Json::objectValue);
+  execution_options["disable_gas_fee"] = false;
+  execution_options["disable_nonce_check"] = true;
+
+  auto &genesis_balances = state["genesis_balances"] = Json::Value(Json::objectValue);
+  auto addr1 = "2cd4da7d3b345e022ca7e997c2bb3276a4d3d2e9";
+  auto addr2 = "7e4aa664f71de4e9d0b4a6473d796372639bdcde";
+  genesis_balances[addr1] = "0x1027e72f1f12813088000000";
+  genesis_balances[addr2] = "0x1027e72f1f12813088000000";
+
+  auto &pbft = chain_config["pbft"] = Json::Value(Json::objectValue);
+  pbft["committee_size"] = "0x64";
+  pbft["dag_blocks_size"] = "0xa";
+  pbft["ghost_path_move_back"] = "0x0";
+  pbft["lambda_ms_min"] = "0x29a";
+  pbft["run_count_votes"] = false;
+
+  auto &vdf = chain_config["vdf"] = Json::Value(Json::objectValue);
+  vdf["difficulty_max"] = "0x12";
+  vdf["difficulty_min"] = "0x10";
+  vdf["threshold_selection"] = "0xbffd";
+  vdf["threshold_vdf_omit"] = "0x6bf7";
+  vdf["difficulty_stale"] = "0x13";
+  vdf["lambda_bound"] = "0x64";
+
+  return chain_config;
+}
+
+Json::Value generateConfigFile(string &db_path, bool boot_node, int network_identifier) {
+  Json::Value config_file(Json::objectValue);
+
+  // Account
+  auto account_file = getAccountFile();
+  if (!fileExist(account_file)) {
+    auto account = dev::KeyPair::create();
+    auto account_json = getAccountJson(account);
+
+    // Create account file
+    auto account_file = getAccountFile();
+    if (!writeFile(account_file, account_json)) {
+      exit(1);
```

Comment:
first of all, EXIT_FAILURE is used instead of 1. And why would you use exit() in some internal function ? Use exception and log error please